### DISCUSSION TITLE
Unchecking "Control Points" in the editor cuts off bottom of the timeline

### DISCUSSION THREAD

**Discussion by joshualomelin** [ppy/osu#16769] [Q&A] [answered]
For some reason, when you toggle the "Control Points" option off at the top left of the editor, it causes the bottom of the entire top section (the one with the timeline) to move up, partially obscuring the labels for "beat snap divisor" and "Ticks"

With Control Points toggled on:
![toggled_on](https://user-images.githubusercontent.com/24350667/152463709-8947317e-abcb-4e88-9425-9c4ebf9300e3.png)

With Control Points toggled off:
![toggled_off](https://user-images.githubusercontent.com/24350667/152463723-d125b419-23c7-4dce-8a26-5ddfd2156632.png)

Scrolling and dragging also does not reveal the obscured labels/buttons.

I am on osu!lazer version 2022.128.0-lazer.

Logs if you need them:
[runtime.log](https://github.com/ppy/osu/files/7999671/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7999672/updater.log)
[database.log](https://github.com/ppy/osu/files/7999673/database.log)
[network.log](https://github.com/ppy/osu/files/7999674/network.log)
[performance.log](https://github.com/ppy/osu/files/7999675/performance.log)

**Comment by peppy** [comment, marked as answer]
This broke way back with the update to checkbox designs, I believe. The "beat snap" being cut off was intentional somewhat, though.

You can temporarily resolve this by adjusting the game's UI scale if it bothers you.

**Reply by hwabis** [reply]
Not sure if this is known, but it's still cut off no matter how small/big I change game UI scale
